testsuite/mlir/mlir-parse/function.verona

```diff
@@ -1,23 +1,22 @@
 // Copyright Microsoft and Project Verona Contributors.
 // SPDX-License-Identifier: MIT
 
-// RUN: verona-mlir %s -o - | FileCheck %s
+empty_declaration();
```

Comment:
Why have you dropped the FileCheck stuff?

Reply:
Because we're not using it, we're comparing directly with the output, like the other tests. This was a left-over from the move that I only did now.